test/test_operators.py

```diff
@@ -286,6 +286,18 @@ def reshape(input):
         ref_out, tvm_out = self.runBoth(reshape, input)
         assert torch.allclose(ref_out, tvm_out, rtol=0.01, atol=0.01)
 
+    @TVMTest.given(
+        shape=TVMTest.rand_shape(rank=2, min_dim=4),
+        axis=TVMTest.rand_int(0, 2),
```

Comment:
whoops, I guess it's supposed to be `rand_int(0, 1)`
(https://circleci.com/gh/pytorch/tvm/428?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)